src/text/2-package-subsections/datasets.tex

```diff
@@ -1,32 +1,12 @@
-
-\begin{table*}[!ht]
-	\centering
-	\begin{tabular}{|l|l|l|l|l|l|l|}
-		\hline
-		Dataset Type         & Data Type & Reduced IRFs                      & Geometry              & Additional Quantities                    & Fit Statistic & Purpose                                                                         \\ \hline
-		MapDataset           & counts    & background, psf, edisp, exposure, & WcsGeom or RegionGeom & --                                       & cash          & A full 3D analysis with a field of view background                              \\ \hline
-		MapDatasetOnOff      & counts    & psf, edisp, exposure              & WcsGeom               & acceptance, acceptance\_off, counts\_off & wstat         & Sky Images with a Ring Background / Background estimation from off observations \\ \hline
-		SpectrumDataset      & counts    & background, edisp, exposure       & RegionGeom            & --                                       & cash          & 1D spectral extraction with the FoV background                                  \\ \hline
-		SpectrumDatasetOnOff & counts    & edisp, exposure                   & RegionGeom            & acceptance, acceptance\_off, counts\_off & wstat         & 1D spectral extractions using Reflected regions                                 \\ \hline
-		FluxPointsDataset    & flux      & None                              & None                  & --                                       & chi2          & Fitting of precomputed flux points                                              \\ \hline
-	\end{tabular}
-	\caption{Some caption.}
-	\label{table:datasets_types}
-\end{table*}
-
-Multiple datasets, or the same or different type, can be contained in a Datasets container. The Datasets class adds the likelihood from its constituent members to compute the full likelihood for a joint fitting.
-
-The total number of predicted counts for a dataset are computed as the sum of the predicted counts from all source components, plus the expected counts from the residual hadronic background. In case of normal datasets, the predicted counts from the hadronic background are computed directly from the model in reconstructed energy and spatial coordinates, whereas for OnOff datasets, the background is estimated from real off counts. The predicted counts from a source are obtained by forward folding with the IRFs, assuming that the IRFs are independent of one another.
```

Comment:
Could keep this for now. We have to talk about the on-off case as well...

Reply:
Thanks @adonath ! I found the first version to verbose. Adapted the writing a bit to keep your comments.